competitions/neurips_2021_docs/docs.css

```diff
@@ -0,0 +1,3 @@
+img.img-fluid, img.thumnail-image {
```

Comment:
where is the `thumnail-image` class used and why 'thumnail' and not 'thumbnail' ?